### DISCUSSION TITLE
temperatuur in woonkamerknop

### DISCUSSION THREAD

**Discussion by etmmast** [dwainscheeren/dwains-lovelace-dashboard#546] [Q&A] [answered]
Welke temperatuur wordt er weergegeven in de woonkamerbutton? er staat bij mij een temperatuur maar die komt niet overeen met een van de bekende temperaturen zoals ketelwater-, ruimtetemperatuur of tapwatertemperatuur of buiten temperatuur

en waar kan ik deze temperatuur instellen, zodat hij de juiste kamertemperatuur weergeeft?

**Comment by etmmast** [comment, marked as answer]
sorry i saw the same question in a another item, also the answer that it will be fixed in version 3.1
gr. Marco

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
